getGeocode.js

```diff
@@ -2,30 +2,32 @@ const dotenv = require("dotenv");
 //note: this only works locally; in Lambda we use environment variables set manually
 dotenv.config();
 
-const fetch = require("node-fetch");
 const { Client } = require("@googlemaps/google-maps-services-js");
 const { generateKey } = require("./data/dataDefaulter");
+const axios = require("axios");
+const axiosRetry = require("axios-retry");
+
+axiosRetry(axios, { retries: 5, retryDelay: axiosRetry.exponentialDelay });
```

Comment:
So I ended up adding the axios retry (Google allows retries in their JavaScript package using axios) and configured an exponential delay. Axios will attempt 5 retries before giving up.

Reply:
I've tried this about 3 times now (starting main.js immediately after the previous execution completes) and so far I haven't gotten a `rate limit exceeded` error yet. 

I suspect this probably won't be much of an issue in production anyways since we are using the previous `data.json`'s lat/long anyways so the geocode API calls shouldn't be too frequent unless a huge amount of new locations get added all at once.